metadata/service/region/single.yml

```diff
@@ -9,6 +9,10 @@ parameters:
       bind:
         host: localhost
         port: 80
+      admin:
+        username: admin
+        password: password
```

Comment:
C'mon.. don't set default passwords in metadata.. some fool will forget it in place..